.github/workflows/golangci-lint.yml

```diff
@@ -26,7 +27,7 @@ jobs:
 
       - uses: actions/setup-go@84cbf8094393cdc5fe1fe1671ff2647332956b1a
         with:
-          go-version: '1.17'
+          go-version: '1.19'
```

Comment:
`1.19.1`

Reply:
Hm, doesn't it grab any 1.19 if that's the case? Or just 1.19.0?

Reply:
No clue, just commented because consistency.

Reply:
I guess it doesn't hurt to make it consistent... agreed.